build.xml

```diff
@@ -846,6 +851,23 @@
 		<antcall target="run"/>
 	</target>
 
+	<!-- If rewrite-frankframework not installed, run installRecipes.bat -->
+	<target name="install-rewrite" unless="rewrite.installed">
```

Comment:
Can you call it install-recipes otherwise it sounds like the rewrite-framework is not present and needs to be installed / cloned